app/config/settings_prod.py

```diff
@@ -70,11 +70,17 @@
     'solo.apps.SoloAppConfig',
     'storages',
     'whitenoise.runserver_nostatic',
+    'django_prometheus',
     'config.apps.StacAdminConfig',
     'stac_api.apps.StacApiConfig',
 ]
 
+# Middlewares are executed in order, once for the incoming
+# request top-down, once for the outgoing response bottom up
+# Note: The prometheus middlewares should always be first and
+# last, put everything else inbetween
 MIDDLEWARE = [
+    'django_prometheus.middleware.PrometheusBeforeMiddleware',
     'middleware.cache_headers.CacheHeadersMiddleware',
```

Comment:
@ltshb just noted that `CacheHeadersMiddleware` is on top of the middleware list. Is this required? If not I would move it all the way down to between `ExceptionLoggingMiddleware` and `PrometheusAfterMiddleware`

Reply:
I don't think this is required, you can try to move it down.

Reply:
done